arbor/cli.py

```
@@ -102,12 +108,66 @@ def stop_server(server):
 @cli.command()
 @click.option("--host", default="0.0.0.0", help="Host to bind to")
 @click.option("--port", default=7453, help="Port to bind to")
-@click.option("--arbor-config", required=True, help="Path to the Arbor config file")
+@click.option("--arbor-config", required=False, help="Path to the Arbor config file")
 def serve(host, port, arbor_config):
     """Start the Arbor API server"""
-    app = create_app(arbor_config)
+    config_path = arbor_config if arbor_config else Settings.find_config_path()
+    create_app(config_path)
```

If no config is found, can you throw an error to tell the user they need to make a config before they start arbor?

---

yep, added config file validation before creating app